thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/dashboard/resources/v2/anomalies/AnomalySearcher.java

```diff
@@ -70,7 +70,11 @@ public AnomalySearcher() {
    * @return the result
    */
   public Map<String, Object> search(AnomalySearchFilter searchFilter, int limit, int offset) {
-    Predicate predicate = Predicate.EQ("child", false);
+    Predicate predicate = Predicate.NEQ("baseId", 0);
```

Comment:
Hi Jihao, is there a way for us to test the fix before waiting for the next deployment? This is a bit critical which is impacting all entity monitoring. Let's make sure it works when we deploy.

Reply:
it will be tested by unit tests and staging before deploy.